docs/Apps/Data Studio/CCLE Correlation Analysis.md

```diff
@@ -1,11 +1,235 @@
+
+Output:
+
+*   **CCLE annotated metadata:** New column will be added which has the information about the mutation status of the selected gene across all the samples. If multiple genes are selected, then multiple columns will be added.
+
+    
+![Annotation output](../../img/ccle/8.png) <center>**Figure 8.** Annotation output</center>
+
+## Filtering
+
+This component is used to filter samples from the expression data and metadata based on your selection. The component provides an option to choose a column and subsequent value from within that column to subset the data.
+
+![Filetering Interface](../../img/ccle/9.png) <center>**Figure 9.** Filtering parameters</center>
+
+*   **Filter samples? :** It provides two options - ‘*Yes*' and '*No*'. If '*No*’ is selected, then no filtering is applied to the data and all the samples are available for usage in the downstream components. If '*Yes*' is selected, then the user must fill-up the options for filtering in the UI elements just beneath this radio button.
+
+*   **Select Column:** Names of the columns from the metadata file are enlisted here for you to choose.
+
+*   **Column Value:** Once a column has been selected in the *Select Column* dropdown, then all the unique values from the selected column are available for you to pick from.
+
+Once all the fields are selected, click on *Run Task* to execute the component.
+
+It will generate three output files:
+
+![Filtering output](../../img/ccle/10.png) <center>**Figure 10.** Filtering output</center>
+
+*   **Expression data:** Subsetted expression data with filtered samples.
+
+*   **Mutation data:** Subsetted mutation matrix with filtered samples.
+
+*   **Metadata:** Subsetted metadata with filtered samples.
+   
+
+# Data Exploration
+
+## Gene expression Box plot
+
+This component generated a gene expression Boxplot. It is a visualization component that uses a five-number summary ( minimum, first quartile, median, third quartile, and maximum) to display the distribution of data based on quartiles. Outliers are also displayed as individual points.
+
+![Box plot interface](../../img/ccle/11.png) <center>**Figure 11.** Boxplot parameters</center>
+
+*   **Select gene:** Dropdown for picking the gene for which the boxplot is to be made.
+
+*   **Select metadata column:** Dropdown to select one of the metadata columns.  
+
+After executing the component, an interactive *Boxplot* will be generated as an output.
+
+![Box plot](../../img/ccle/12.png) <center>**Figure 12.** Boxplot</center>
+    
+
+## **PCA**
+
+This component allows you to simplifies the complexity of high-dimensional data while retaining the trends and patterns in it. It projects the data onto a lower dimension with an objective to find the best summary of the data using a limited number of principal components that help in understanding the clustering pattern between biologically grouped and ungrouped samples.
```

Comment:
keep this consistent with the PCA definition on the platform? also `simplify` and not `simplifies`